api/container/containerv1/clusters.go

```diff
@@ -31,6 +31,20 @@ type ClusterInfo struct {
 	State             string
 	IsPaid            bool
 	WorkerCount       int
+	Vlans             []Vlan
+}
+
+type Vlan struct {
+	ID      string
+	Subnets []SubnetV
+}
+
+type SubnetV struct {
+	CIDR     string
```

Comment:
How about just Cidr, Id, Ips?

Reply:
Id puting it as ID . Changed Cidr and Ips